packages/gatsby-theme-apollo-docs/src/components/page-content.js

```diff
@@ -110,6 +110,9 @@ const AsideLinkInner = styled.a({
   }
 });
 
+const makeGithubUrl = (host, repo) => host ? `https://${host}/${repo}` : repo;
```

Comment:
It seems like the work being done here is not necessary if you're creating `githubUrl` in `gatsby-node.js`. You can also avoid passing `githubHost` as a prop to `PageContext` within the `Template` component.